gulpfile.js

```diff
@@ -3,7 +3,9 @@ var gulpif = require('gulp-if');
 var cleanCSS = require('gulp-clean-css');
 var del = require('del');
 var dom = require('gulp-dom');
-var uglify = require('gulp-uglify');
+var uglifyes = require('uglify-es');
+var composer = require('gulp-uglify/composer');
+var uglify = composer(uglifyes, console);
```

Comment:
what this compositing does?

Reply:
This is the method provided by `gulp-uglify` to replace the version of `uglifyjs` that it uses internally. For reference: https://github.com/terinjokes/gulp-uglify#using-a-different-uglifyjs